### ISSUE TITLE
The script returns error for usb.core

### ISSUE THREAD

**Issue by variable-enigmatic** [joseluis/huion-linux-drivers#76]
Running the script results in this:

`Traceback (most recent call last):
  File "./huion-tablet-driver.py", line 8, in <module>
    import usb.core
ModuleNotFoundError: No module named 'usb'
`
I believe that this is because I have multiple versions of Python installed. My system apparently installs the module pyusb into Python2.7. I'm not sure what to do about this.

**Comment by joseluis** [comment]
Yeah, many problems are arising related to different versions of python and different versions of linux distributions which are appearing now, and also betwen display servers (x11 / wayland)... I got fed trying to solve them and I'm currently rewriting the program in Rust and it will be much more reliable. I hope to have a beta this month.
